source/SkiaSharp.Views.Uno/SkiaSharp.Views.Uno.Mac/SKXamlCanvas.macOS.cs

```diff
@@ -20,6 +20,9 @@ public partial class SKXamlCanvas
 		public SKXamlCanvas()
 		{
 			Initialize();
+#if MACOS
+			Opaque = false;
+#endif
```

Comment:
This is actually incorrect for macOS. There is no `Opaque` property, but rather `IsOpaque`. However, this other property is read only and according to the docs you should override the property to return the value you want:

> The default value of this property is false to reflect the fact that views do no drawing by default. Subclasses can override this property and return true to indicate that the view completely covers its frame rectangle with opaque content. Doing so can improve performance during drawing operations by eliminating the need to render content behind the view.
> https://developer.apple.com/documentation/appkit/nsview/1483558-isopaque

Reply:
Fixed in https://github.com/mono/SkiaSharp/pull/2477

Reply:
oops sorry about that ! I thought I tested it :/